server/firebase.js

```diff
@@ -1,18 +1,22 @@
-const { initializeApp, cert } = require("firebase-admin/app");
-const { getFirestore, FieldValue,  Timestamp, } = require("firebase-admin/firestore");
-const { getAuth } = require("firebase-admin/auth");
+const {initializeApp, cert} = require("firebase-admin/app");
+const {
+  getFirestore,
+  FieldValue,
+  Timestamp,
+} = require("firebase-admin/firestore");
+const {getAuth} = require("firebase-admin/auth");
 const admin = require("firebase-admin");
 
 const serviceAccount = require("./credentials.json");
 
 const app = initializeApp({
   credential: cert(serviceAccount),
-  storageBucket: process.env.REACT_APP_STORAGEBUKET,
```

Comment:
the address is not good here no ?

Reply:
I use storage bucket in backlog ressources, can you re-fix that and readd in const app storageBucket